parallel-install/pkg/installation/installation.go

```diff
@@ -15,25 +17,40 @@ import (
 )
 
 type Installation struct {
-	// Map component > namespace
+	// Slice of pairs: [component, namespace]
 	Prerequisites [][]string
 	// Content of the Installation CR YAML file
 	ComponentsYaml string
 	// Content of the Helm overrides YAML files
 	OverridesYamls []string
-	ResourcesPath  string
-	Cfg            config.Config
+	// Rood dir in local filesystem with subdirectories containing components' Helm charts
+	ResourcesPath string
```

Comment:
You used `rootDir` in ComponentsProvider and PrerequisitesProvider. Maybe it could be unified?

Reply:
👍

Reply:
That would require changes to CLI. I'd like to avoid at the moment. This PR was not intended to change the API.

Reply:
OK, I changed `rootDir` -> `resourcesPath` so it's the same across the library.